src/pages/detailedNotice/FullPageImageViewer.tsx

```diff
@@ -0,0 +1,134 @@
+import { useHotkeys } from "@mantine/hooks";
+import { useState } from "react";
+import { Close, Download } from "src/assets/Icons";
+import Flex from "src/atoms/containers/flex/Flex";
+import Image from "src/atoms/image/Image";
+import Spacer from "src/atoms/spacer/Spacer";
+import Text from "src/atoms/text/Text";
+import colorSet from "src/styles/colorSet";
+import Font from "src/styles/font";
+import styled from "styled-components";
+
+import LongArrowSvg from "./assets/LongArrowSvg";
+
+interface FullPageImageViewerProps {
+  srcs: string[];
+  startIndex?: number;
+  onClose?: () => void;
+}
+
+const FullPageWrapper = styled(Flex)`
+  width: 100vw;
+  height: 100vh;
+
+  z-index: 100;
+  position: fixed;
+  top: 0;
+  left: 0;
+
+  background-color: rgba(0, 0, 0, 0.6);
+`;
+
+const ImageNavigation = styled(Flex)`
+  background-color: ${colorSet.text};
+  padding: 5px;
+`;
+
+const MenuSum = styled(Flex)`
+  position: absolute;
+  top: 10px;
+  right: 20px;
+`;
+
+const Menu = styled(Flex)`
+  cursor: pointer;
+`;
+
+const FullPageImageViewer = ({
+  srcs,
+  startIndex,
+  onClose,
+}: FullPageImageViewerProps) => {
+  const [imageIndex, setImageIndex] = useState<number>(startIndex ?? 0);
+  const maxIndex = srcs.length - 1;
+
+  const ManipulateIndex = (amount: number) => {
+    if (imageIndex + amount < 0 || imageIndex + amount > maxIndex) return;
+    setImageIndex((imageIndex) => imageIndex + amount);
```

Comment:
저는 Math 함수를 써서 이렇게 하기도 해요


```suggestion
    setImageIndex((imageIndex) => Math.max(0, Math.min(imageIndex + amount, maxIndex)));
```

Reply:
깔끔하고 좋네여

Reply:
배너 체인지 인덱싱에도 적용했습니다